modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/resource/v1_0/test/RelatedObjectEntryResourceTest.java

```diff
@@ -217,6 +289,10 @@ private JSONObject _invoke(String endpoint, Http.Method httpMethod)
 			options.setPut(true);
 		}
 
+		if (httpMethod == Http.Method.DELETE) {
+			options.setDelete(true);
```

Comment:
Instead of setting the specific method with the true value, we could replace with options.setMethod(httpMethod); both invocations of the PUT and DELETE if clauses